tools/jinja2-filters.py

```diff
@@ -0,0 +1,19 @@
+import base64
+
+class SecretIsNotString(Exception):
+    pass
+
+class NotSupportedDataType(Exception):
+    pass
+
+def filter_b64decode(secret_value, data_type = "string"):
+    if not isinstance(secret_value, str):
+        raise SecretIsNotString
+    if data_type == "string":
+        # The code above return exception:
+        # binascii.Error if secret_value is not correct base64 data
+        # UnicodeDecodeError if you try decode bytes to utf-8 format
+        return base64.b64decode(secret_value).decode("utf-8")
```

Comment:
> The code above

You mean "code below'.

Please reword this:
``` python
# If the input is invalid, decoding can result in the following exceptions:
# - binascii.Error     - if the input is not base64 encoded
# - UnicodeDecodeError - if the encoded value is not a valid UTF-8 string
```

Reply:
Done.